accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/rest/controllers/BasicAccessionHistoryController.java

```diff
@@ -44,4 +46,10 @@ public BasicAccessionHistoryController(HistoryService<MODEL, ACCESSION> service,
         return converter.convert(service.getHistory(accession));
     }
 
+    @RequestMapping(value = "/{accession}", method = RequestMethod.GET, produces = "application/json")
+    public AccessionWrapper<MODEL, String, ACCESSION> getAccessionMergedInto(@PathVariable ACCESSION accession) throws
```

Comment:
How do you return "Redirection" codes etc. if an accession has been merged into another one?

Reply:
if its merged it returns the merged accession , if not it gives appropriate exceptions

Reply:
Yes, I can see what is being returned here. But I am not sure if that is what is required. Do we expect to return a 3xx code and a redirect URI, a 3xx code and the accession merged into, a 200 and the accession merged into or anything else? @cyenyxe

Reply:
Ya understand your question now ,i can build  redirect uri but all endpoints returns response codes only on errors. I am not sure if i need to . @cyenyxe is redirect uri a must here or just the accession is fine ?